loopchain/channel/channel_inner_service.py

```diff
@@ -601,6 +602,32 @@ async def get_status(self):
     async def make_essential_backup(self, block_height):
         return await BackupManager().make_backup(self._blockchain, block_height)
 
+    @message_queue_task(priority=255)
+    async def block_height_request(self, request_from: str):
+        """Handle BlockHeightRequest.
+
+        Responses last height of this node.
+        """
+        last_block = self._channel_service.block_manager.blockchain.last_block
+        height: int = last_block.header.height
+        response_type = loopchain_pb2.HeightResponse(
+            peer=ChannelProperty().peer_target,
+            channel=ChannelProperty().name,
+            height=height
+        )
+
+        channel = GRPCHelper().create_client_channel(request_from)
+        stub = loopchain_pb2_grpc.PeerServiceStub(channel)
+        stub.BlockHeightResponse(response_type, conf.GRPC_TIMEOUT_SHORT)
```

Comment:
Awaiting helpful reviews:
- how to test this?
- send response here or somewhere else?